README.md

```diff
@@ -89,9 +89,41 @@ deepstate-angr --num_workers 4 --output_test_dir out $DEEPSTATE/build/examples/I
 
 DeepState consists of a static library, used to write test harnesses, and command-line _executors_ written in Python. At this time, the best documentation is in the [examples](/examples) and in our [paper](https://agroce.github.io/bar18.pdf).
 
-## Fuzzing
+## Fuzzing with libFuzzer
 
-DeepState now can be used with a file-based fuzzer (e.g. AFL).  There
+If you install clang 6.0 or later, and run `cmake` when you install
+with the `BUILD_LIBFUZZER` environment variable defined, you can
+generate tests using LlibFuzzer.  Because both DeepState and libFuzzer
+want to be `main`, this requires building a different executable for
+libFuzzer.  The `examples` directory shows how this can be done.  The
+libFuzzer executable works like any other libFuzzer executable, and
+the tests produced can be run using the normal DeepState executable.
+For example, generating some tests of the `OneOf` example (up to 5,000
+runs), then running those tests to examine the results, would look
+like:
+
+```shell
+mkdir OneOf_libFuzzer_corpus
+./OneOf_LF OneOf_libFuzzer_corpus -runs 5000
```

Comment:
This line only works for me like this `./OneOf_LF -runs=5000 OneOf_libFuzzer_corpus`. I'm using clang-6.0 (maybe other versions are more flexible with the command line arguments)